pyatlan/model/assets/dbt_process.py

```diff
@@ -536,6 +551,9 @@ class Attributes(Dbt.Attributes):
         airflow_tasks: Optional[List[AirflowTask]] = Field(
             default=None, description=""
         )  # relationship
+        alteryx_workflow: Optional[AlteryxWorkflow] = Field(
+            default=None, description=""
+        )  # relationship
```

Comment:
This is a one-off customization (in-progress experiment) — we should remove from generated code.